components/Layout/AudioControls.vue

```diff
@@ -1,135 +1,315 @@
+                </v-btn>
+                <v-btn
+                  elevation="6"
+                  fab
+                  icon
+                  raised
+                  rounded
+                  outlined
+                  class="mx-1"
+                  @click="togglePause"
+                >
+                  <v-icon large>
+                    {{ isPaused ? 'mdi-play' : 'mdi-pause' }}
+                  </v-icon>
+                </v-btn>
+                <v-btn
+                  icon
+                  :disabled="!getNextItem"
+                  class="mx-1"
+                  @click="setNextTrack"
+                >
+                  <v-icon>mdi-skip-next</v-icon>
+                </v-btn>
+                <v-btn
+                  icon
+                  fab
+                  small
+                  :elevation="isRepeating ? '3' : '0'"
+                  class="mx-1"
+                  :class="isRepeating ? '' : 'text--disabled'"
+                  :color="isRepeating ? 'primary' : undefined"
+                  @click="toggleRepeatMode"
+                >
+                  <v-icon>{{ repeatIcon }}</v-icon>
+                </v-btn>
+              </div>
+              <time-slider v-if="!isFullScreenPlayer" />
+            </div>
+          </v-col>
+          <v-col cols="3" class="d-none d-md-flex align-center justify-end">
             <v-btn class="d-none d-md-inline-flex" icon disabled>
               <v-icon size="18">{{
                 getCurrentItem.UserData.IsFavorite
                   ? 'mdi-heart'
                   : 'mdi-heart-outline'
               }}</v-icon>
             </v-btn>
-          </span>
-          <nuxt-link
-            v-if="getCurrentItem.AlbumArtists[0].Id"
-            tag="span"
-            class="text--secondary text-caption text-truncate mt-md-n2 link"
-            :to="`/artist/${getCurrentItem.AlbumArtists[0].Id}`"
+            <v-tooltip top>
+              <template #activator="{ on, attrs }">
+                <v-btn disabled icon class="mr-2" v-bind="attrs" v-on="on">
+                  <v-icon>mdi-playlist-play</v-icon>
+                </v-btn>
+              </template>
+              <span>{{ $t('queue') }}</span>
+            </v-tooltip>
+            <volume-slider />
+            <transition name="fade-fast" mode="in-out">
+              <v-tooltip v-if="!isFullScreenPlayer" top>
+                <template #activator="{ on, attrs }">
+                  <nuxt-link tag="span" :to="'/playback'">
+                    <v-btn icon class="ml-2" v-bind="attrs" v-on="on">
+                      <v-icon>mdi-fullscreen</v-icon>
+                    </v-btn>
+                  </nuxt-link>
+                </template>
+                <span>{{ $t('fullScreen') }}</span>
+              </v-tooltip>
+            </transition>
+            <v-tooltip top>
+              <template #activator="{ on, attrs }">
+                <v-btn
+                  icon
+                  class="ml-2"
```

Comment:
```suggestion
```

Currently looks slightly off with the enlarged gap between icons

![image](https://user-images.githubusercontent.com/18101008/104507042-1ba79400-55de-11eb-97e1-8cd0ba0a1093.png)